xml/System.Data/ParameterDirection.xml

```diff
@@ -152,7 +152,7 @@
       </ReturnValue>
       <MemberValue>2</MemberValue>
       <Docs>
-        <summary>The parameter is an output parameter.</summary>
+        <summary>The parameter is an output parameter.  This is useful when a stored procedure returns data rows and it is necessary to return additional values such as a sum total or number of errors.  An output parameter is different from a return value.  A stored procedure can use an output parameter and a return value to return different data.</summary>
```

Comment:
It's important to remember that these types are database-agnostic, and so we need to be careful not to include any database-specific information; in many databases, there's no distinction between output parameters and return values (indeed, the concepts themselves don't exist in many cases). So it's better not to go into too much detail. Output values are also used in various scenarios where the sproc doesn't return data rows at all.

```suggestion
        <summary>The parameter is an output parameter, which can be set by a stored procedure or function to return a value. This value is separate from and can be returned in addition to any data rows.</summary>
```